### DISCUSSION TITLE
Newsletter system

### DISCUSSION THREAD

**Discussion by duartecancela** [timlrx/tailwind-nextjs-starter-blog#500] [Q&A] [answered]
Hi, the newsletter is module is working?  I changed my provider to mailchimp and added envoirment variables, but when I submit give me an error to console, ' you must enter an email', can someone help-me?

**Comment by dlarroder** [comment, marked as answer]
@duartecancela I've cloned your [repo](https://github.com/duartecancela/nextjs-digitaldeveloper) and added my personal `env` variables and it worked for me. Video below:

https://user-images.githubusercontent.com/52998821/177998016-495b2b00-6679-4cf2-b944-963c7c0a826f.mov

The form is working perfectly, I suspect it might be your env variables?

Here are some useful links on where to find your Mailchimp API keys

* [Link](https://www.youtube.com/watch?v=6jKpsVWrTLQ) for getting `MAILCHIMP_API_KEY` and `MAILCHIMP_AUDIENCE_ID`
* [Link](https://stackoverflow.com/questions/64814642/mailchimp-integration-with-php)  for getting `MAILCHIMP_API_SERVER`

Hope this helps!

**Reply by duartecancela** [reply]
Thank you very much! I will check my envoirment variables 👍
